wheelSeparation: 0.34
Frequency: 10.0
timeout: 0.5
minFreq: 0.2
maxFreq: 10.0

# Controller Type: 0->Decoupled, 1->Coupled
controllerType: 0


# Controller_1

maxVel1: 0.70
integralConst1: 1.0
derivativeConst1: 0.4
proportionalConst1: 0.8
windupConst1: 0.6
cntr1Name: LeftController


# Controller_2

maxVel2: 0.70
integralConst2: 1.0
derivativeConst2: 0.4
proportionalConst2: 0.8
windupConst2: 0.6
cntr2Name: RightController
